content/en/synthetics/browser_tests/actions.md

```diff
@@ -13,106 +13,139 @@ further_reading:
 
 ## Overview
 
-Steps are a series of actions that you can record for a browser test, which you can then edit or build on. You can define the steps you want your browser test to go through either by directly recording them with the Datadog test recorder extension or by manually adding the step of interest. All steps come with a set of [advanced options][1] that you can configure.
+Steps are a series of actions that you can record for a browser test and edit or build on. To define the steps you want your browser test to execute, either directly record them with the Datadog test recorder extension or add them manually. Every step includes a set of configurable [advanced options][1].
 
-**Note**: The default timeout for each step is approximately 60 seconds. You can override this default timeout through the dedicated [advanced option][2].
+The default timeout for each step is approximately 60 seconds. You can override this default timeout through the dedicated [timeout option][2]. 
+
+## Manage step order
```

Comment:
Why should it be the first section of the page?

Reply:
move this above use variables section